orchagent/pfcactionhandler.h

```diff
@@ -0,0 +1,106 @@
+#ifndef PFC_ACTION_HANDLER_H
+#define PFC_ACTION_HANDLER_H
+
+#include <vector>
+#include <memory>
+
+#include "table.h"
+
+extern "C" {
+#include "sai.h"
+}
+
+using namespace std;
+using namespace swss;
+
+// PFC queue interface class
+// It resembles RAII behavior - pause storm is mitigated (queue is locked) on creation,
+// and is restored (queue released) on removal
+class PfcWdActionHandler
+{
+    public:
+        PfcWdActionHandler(sai_object_id_t port, sai_object_id_t queue,
+                uint8_t queueId, shared_ptr<Table> countersTable);
+        virtual ~PfcWdActionHandler(void) = 0;
+
+        inline sai_object_id_t getPort(void)
```

Comment:
It should be fine to remove inline, as they are inline by default.

Reply:
I know that compiler will optimize that anyway, but is this behavior defined by a standard?

Reply:
As I remembered, the functions defined in the class are inline by default.